cw/capture_sha3_cw310.yaml

```diff
@@ -8,24 +8,25 @@ capture:
   key_len_bytes: 16
   plain_text_len_bytes: 16
   output_len_bytes: 32
+  # Switch the masking on or off - Works for SHA3 only. Doesn't work when
+  # processing key material.
   masks_off: false
-  # Samples per trace - We oversample by 20x
-  num_samples: 3500
-  # Offest in samples
-  offset: 0
-  # 32-bit seed for SW key masking. Key masks are generated using an LFSR.
-  # For unprotected implemetation, lfsr_seed should be set to 0. This will
-  # effectively switch off the masking. For masked implementation, any seed
-  # other than 0 should be used.
-  # w/o DOM
-  #lfsr_seed: 0
-  # w/ DOM
+  # Samples per trace - We oversample by 20x and SHA3 with DOM is doing 120
+  # cycles (24 for loading and padding, 96 for processing) with 320 delay
+  # cycles between loading the plaintext and adding the padding. The plaintext
+  # loading and the delay cycles can be ignored.
```

Comment:
Should this comment be added to `capture_sha3_masks_off_cw310.yaml`

Reply:
Oh, I wasn't aware we had a separate config file. Let me align the two. Thanks for pointing this out!